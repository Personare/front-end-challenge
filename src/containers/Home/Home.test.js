describe('A simple test', () => {
    it('Should return true', () => {
        expect(5).toBeGreaterThan(2)
    })
})
