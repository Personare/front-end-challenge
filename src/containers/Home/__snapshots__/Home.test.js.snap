// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly 1`] = `
<div
  className="sc-htpNat KtFsv"
>
  <button
    onClick={[Function]}
  >
    Iniciar jogo
  </button>
  <div
    className="sc-bxivhb dehkWE"
  />
</div>
`;
